lib/user_space/user_x86_64_atomic.h

```diff
@@ -31,15 +31,19 @@
 /**
    @addtogroup atomic
 
-   Implementation of atomic operations for Linux user space uses x86_64 assembly
-   language instructions (with gcc syntax). "Lock" prefix is used
-   everywhere---no optimisation for non-SMP configurations in present.
+   Implementation of atomic operations for Linux user space uses x86_64 
+   assembly language instructions (with gcc syntax). "Lock" prefix is used
+   everywhere for x86_64 platform assembely instruction to ensure atomicity 
```

Comment:
Remove "for x86_64 platform assembely instruction" - it's clear from the previous sentence that we are talking about x86_64 here. No need to repeat it again.

Reply:
The comment was there as before, why to remove ?

Reply:
I don't ask to remove all the comment, but only the part of it which I quoted: "for x86_64 platform assembely instruction". And no - this part was not there before. You can compare with the old text.

Reply:
Fixed

Reply:
It's still not fixed. You left "instruction" word there, which makes the sentence even more confusing.

Reply:
Fixed